R/utils.R

```diff
@@ -44,3 +44,13 @@ redirect_id <- function(id, url) {
   url <- header$url
   drive_extract_id(url)
 }
+
+get_mime_type <- function(x) {
+  if (!(x %in% c("document", "spreadsheet", "folder", "presentation",
+                 "form", "drawing", "script"))) {
+    message(glue::glue("Input was not a Google Drive type: {x}"))
+    invisible(NULL)
+  } else {
+  paste0("application/vnd.google-apps.", x)
+  }
+}
```

Comment:
Let's split this into a separate PR? I think it should be doing a lookup, just on general principle, but I confess I can't talk you why we can't do the above for now. Except, for example, I think if you put a real mime type in, it should pass through unharmed.

Again, let's do this in a different branch/PR.